content/en/docs/setup/_index.md

```diff
@@ -12,84 +12,83 @@ content_template: templates/concept
 
-Deciding where to run Kubernetes depends on what resources you have available 
-and how much flexibility you need. You can run Kubernetes almost anywhere, 
-from your laptop to VMs on a cloud provider to a rack of bare metal servers. 
-You can also set up a fully-managed cluster by running a single command or craft 
-your own customized cluster on your bare metal servers.
+You can create a Kubernetes cluster in learning and production environments.
 
 {{% /capture %}}
 
 {{% capture body %}}
 
-## Local-machine Solutions
+## Learning environment
+
+If you want to get familiar with Kubernetes, you can use the Docker-based solutions. You can use tools that are supported by the Kubernetes community or the tools in the ecosystem to set up a Kubernetes cluster on a local machine.
+
+{{< table caption="Local machine solutions table that lists the tools supported by the community and the ecosystem to deploy Kubernetes." >}}
+
+|Community           |Ecosystem     |
+| ------------       | --------     |
+| [Minikube](/docs/setup/minikube/) | [CDK on LXD](https://www.ubuntu.com/kubernetes/docs/install-local) |
+| [Kubeadm-dind](https://github.com/kubernetes-sigs/kubeadm-dind-cluster) | [Docker Desktop](https://www.docker.com/products/docker-desktop)|
+| [Kubernetes IN Docker](https://github.com/kubernetes-sigs/kind) | [Minishift](https://docs.okd.io/latest/minishift/)|
+|                     | [MicroK8s](https://microk8s.io/)|
+|                     | [IBM Cloud Private-CE (Community Edition)](https://github.com/IBM/deploy-ibm-cloud-private) |
+|                     | [IBM Cloud Private-CE (Community Edition) on Linux Containers](https://github.com/HSBawa/icp-ce-on-linux-containers)|
+|                     | [k3s](https://k3s.io)|
+|                     | [Ubuntu on LXD](/docs/getting-started-guides/ubuntu/)|
+
+
+## Production environment
+
+Some of the possible abstractions of a Kubernetes cluster are {{< glossary_tooltip text="applications" term_id="applications" >}}, {{< glossary_tooltip text="data plane" term_id="data-plane" >}}, {{< glossary_tooltip text="control plane" term_id="control-plane" >}}, {{< glossary_tooltip text="cluster infrastructure" term_id="cluster-infrastructure" >}}, and {{< glossary_tooltip text="cluster operations" term_id="cluster-operations" >}}.
+This production environment solutions diagram lists the possible abstractions of a Kubernetes cluster and the level of expertise involved in managing the operations of a cluster.
+
+Production environment solutions![Production environment solutions](/images/docs/KubernetesSolutions.svg)
+
+{{< table caption="Production environment solutions table lists the providers and the solutions." >}}
+Production environment solutions table lists the providers and the solutions that they offer.
```

Comment:
"This" or "The folllowing" production environment solutions table ...

Reply:
I am sorry I missed it . Updated it now.